src/networks.json

```diff
@@ -1409,5 +1409,20 @@
     "explorer": "https://visionscan.org",
     "start": 75909,
     "logo": "ipfs://QmXgGmxDAW2Mheum3WX7Q52Zi9hvE17Zp1pVtZbcVdThh4"
+  },
+  "12321": {
+    "key": "12321",
+    "name": "Wennect Testnet",
+    "shortName": "Wennect",
+    "testnet": true,
+    "chainId": 12321,
+    "network": "testnet",
+    "multicall": "0x009c55698516Fe3C58105Fe9bBC5A33ECE7A92e4",
+    "rpc": [
+      "https://rpc.testnet.wennect.com"
```

Comment:
Looks like this is not a full archive node? i get this error:

```
missing trie node 09e6bd9fea168ca7c6de0a4e06dde3f10f80609970bdb5d6717d63ecb19a64de (path
```

Snapshot need a full archive node

Reply:
@ChaituVR could you please look into it now? It should be full archive node now.